proxy/envoy/testdata/envoy.json.golden

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
Either would work. Maybe just add a comment if you chose to reuse the existing "go test && cp" method?